lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,32 @@
+<div id="change-session-<%-entitlementUUID%>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('Change to a different session or leave current session.')%>
+        <% } else { %>
+            <%- gettext('To access the course, select a session.')%>
+        <% } %>
+    </div>
+    <div class="action-controls">
+        <select class="session-select" aria-label="<%- gettext('Session Selection Dropdown')%>">
```

Comment:
It would be great if you could add the course title here for context, i.e. Session Selection Dropdown for Sample Course A

Reply:
Changed